networkx/algorithms/matching.py

```diff
@@ -329,6 +359,12 @@ def max_weight_matching(G, maxcardinality=False, weight="weight"):
     matching : set
         A maximal matching of the graph.
 
+     Examples
+    --------
+    >>> G = nx.Graph ([(1, 2), (1, 3), (2, 3), (2, 4), (3, 5), (4, 5)])
+    >>> sorted(nx.max_weight_matching(G))
+    [(4, 2), (5, 3)]
+
```

Comment:
Since this is an example for max *weighted* matching it would be more compelling if the example graph included edge weights.

Reply:
Okay sir, thanks, I will re adjust the example